static/css/index.css

```diff
@@ -402,3 +410,114 @@ header {
 }
 
 /* end content styles */
+
+/* start media query styles */
+@media screen and (min-width: 700px) {
```

Comment:
- [ ] As the last code reviewer said 👀: **The value for your min-width should be within the range of 768px** instead of 700px.

Reply:
- Thanks for the review 😊I appreciate that I will do these changes, have a good day ❤️